pkg/storage/gcp/checker.go

```diff
@@ -14,5 +14,5 @@ func (s *Storage) Exists(ctx context.Context, module, version string) (bool, err
 	const op errors.Op = "gcp.Exists"
 	ctx, span := observ.StartSpan(ctx, op.String())
 	defer span.End()
-	return s.bucket.Exists(ctx, config.PackageVersionedName(module, version, "mod"))
+	return s.exists(ctx, config.PackageVersionedName(module, version, "mod"))
```

Comment:
same as `Delete`, we should make sure this is getting wrapped with `errors.E(op, err)`

Reply:
isn't it interested that `const op` is not being used yet the go compiler doesn't give you an unused var error? not sure why consts are an exception when they're scoped.